java/src/main/java/com/google/oak/client/BUILD

```diff
@@ -22,29 +22,15 @@ package(
 )
 
 java_library(
-    name = "client",
+    name = "oak_client",
```

Comment:
Usually the main library or binary is named the same as the containing folder; though I guess here you wanted to name this after the contained class, right? Anyways, up to you

Reply:
Yes, I name Bazel rules after the corresponding class, so it's easier to remember.